pkg/chart/chart.go

```diff
@@ -309,7 +309,7 @@ func (c *Chart) InitNetClient(details *Details) (HTTPClient, error) {
 		namespace := os.Getenv("POD_NAMESPACE")
```

Comment:
@andresmgot So GH doesn't allow commenting on the lines not in the context, it seems. Anyway, up above on line 301, do you remember in what scenarious the call to `SystemCertPool()` would fail (that we are ignoring and just creating an empty pool)? I'm keen to add a handler for that error, but don't know if it's necessary for some situation?

Reply:
If I remember correctly is kind of a corner case. In case there is no system certs, a user may still want to go ahead and use just their CA. `SystemCertPool` returns an error if it's not able to find any cert.